<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>metaItemProperties</class>
 <widget class="QDialog" name="metaItemProperties">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>452</width>
    <height>512</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Connected Items</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Rearrange connected items, unchecking will break the connection.</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSplitter" name="plotListSplitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Connected items</string>
          </property>
          <property name="buddy">
           <cstring>connectedItemsList</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="connectedItemsList">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::ContiguousSelection</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QToolButton" name="moveUpButton">
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="moveDownButton">
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="dataPointsLabel">
          <property name="text">
           <string>Data points</string>
          </property>
          <property name="buddy">
           <cstring>dataTable</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTableWidget" name="dataTable">
          <property name="columnCount">
           <number>2</number>
          </property>
          <attribute name="horizontalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <column>
           <property name="text">
            <string>x</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>y</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QwtPlot" name="itemPreview"/>
       </item>
       <item>
        <widget class="QwtPlot" name="metaPlot"/>
       </item>
       <item>
        <widget class="QCheckBox" name="styleFit">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Styling operations apply to fit</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QwtPlot</class>
   <extends>QFrame</extends>
   <header>qwt_plot.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>metaItemProperties</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>482</x>
     <y>307</y>
    </hint>
    <hint type="destinationlabel">
     <x>202</x>
     <y>301</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>metaItemProperties</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>492</x>
     <y>311</y>
    </hint>
    <hint type="destinationlabel">
     <x>292</x>
     <y>307</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>selectionChanged()</slot>
 </slots>
</ui>
